plutus-benchmark/nofib/src/PlutusBenchmark/NoFib/Clausify.hs

```diff
@@ -63,108 +62,108 @@ disin (Dis p q) =
   dq = disin q
 disin (Con p q) = Con (disin p) (disin q)
 disin p = p
+{-# INLINABLE disin #-}
 
 -- split conjunctive proposition into a list of conjuncts
-{-# INLINABLE split #-}
 split :: Formula -> [Formula]
 split p = split' p []
           where
           split' (Con p q) a = split' p (split' q a)
           split' p a         = p : a
+{-# INLINABLE split #-}
 
 -- eliminate connectives other than not, disjunction and conjunction
-{-# INLINABLE elim #-}
 elim :: Formula -> Formula
 elim (Sym s)    = Sym s
 elim (Not p)    = Not (elim p)
 elim (Dis p q)  = Dis (elim p) (elim q)
 elim (Con p q)  = Con (elim p) (elim q)
 elim (Imp p q)  = Dis (Not (elim p)) (elim q)
 elim (Eqv f f') = Con (elim (Imp f f')) (elim (Imp f' f))
+{-# INLINABLE elim #-}
 
 -- insertion of an item into an ordered list
 -- Note: this is a corrected version from Colin (94/05/03 WDP)
-{-# INLINABLE insert #-}
 insert :: (Ord t) => t -> [t] -> [t]
 insert x [] = [x]
 insert x p@(y:ys) =
   if x < y then x : p
   else if x > y then y : insert x ys
   else p
+{-# INLINABLE insert #-}
 
 -- shift negation to innermost positions
-{-# INLINABLE negin #-}
 negin :: Formula -> Formula
 negin (Not (Not p))   = negin p
 negin (Not (Con p q)) = Dis (negin (Not p)) (negin (Not q))
 negin (Not (Dis p q)) = Con (negin (Not p)) (negin (Not q))
 negin (Dis p q)       = Dis (negin p) (negin q)
 negin (Con p q)       = Con (negin p) (negin q)
 negin p               = p
+{-# INLINABLE negin #-}
 
 -- does any symbol appear in both consequent and antecedent of clause
-{-# INLINABLE tautclause #-}
 tautclause :: LRVars -> Bool
 tautclause (c,a) = [x | x <- c, x `elem` a] /= []
+{-# INLINABLE tautclause #-}
 
 -- form unique clausal axioms excluding tautologies
-{-# INLINABLE unicl #-}
 unicl :: [Formula] -> [LRVars]
 unicl a = foldr unicl' [] a
           where
           unicl' p x = if tautclause cp then x else insert cp x
                        where
                        cp = clause p :: LRVars
+{-# INLINABLE unicl #-}
 
-{-# INLINABLE while #-}
 while :: (t -> Bool) -> (t -> t) -> t -> t
 while p f x = if p x then while p f (f x) else x
+{-# INLINABLE while #-}
 
-{-# INLINABLE formula1 #-}
 formula1 :: Formula  -- % (a = a) = (a = a) = (a = a)
 formula1 = Eqv (Eqv (Sym 1) (Sym 1))
                (Eqv (Eqv (Sym 1) (Sym 1))
                     (Eqv (Sym 1) (Sym 1)))
+{-# INLINABLE formula1 #-}
 
-{-# INLINABLE formula2 #-} -- % One execution takes about 0.35s and 300 MB
 formula2 :: Formula  -- (a = a = a) = (a = a = a)
 formula2 = Eqv (Eqv (Sym 1) (Eqv (Sym 1) (Sym 1)))
                (Eqv (Sym 1) (Eqv (Sym 1) (Sym 1)))
+{-# INLINABLE formula2 #-} -- % One execution takes about 0.35s and 300 MB
```

Comment:
Its peculiar: the comment moved together with the pragma. I don't think this is what we want.